src/crrp-notification-worker/src/main/java/ca/bc/gov/open/pssg/rsbc/crrp/notification/worker/Keys.java

```diff
@@ -0,0 +1,17 @@
+package ca.bc.gov.open.pssg.rsbc.crrp.notification.worker;
+
+public class Keys {
+
+    private Keys() {}
+
+    public static final String CRRP_VALUE = "CRRP";
```

Comment:
should line 7 and line 9 just exist together?

Reply:
no keys is to prevent anyone to create an instance of the Keys class